File: lab/python/numeros_sequencia/sequencias.py
def pares_ordenados(lista):
    i = 0
    ordenados = []
    for i in range(len(lista)):
        for j in range(i + 1, len(lista)):
            ordenados.append((lista[i], lista[j]))
    return ordenados


if __name__ == "__main__":
    lista = [1, 2, 3, 4]
    print(pares_ordenados(lista))

File: lab/javascript/numeros_sequencia/sequencias.js
function pares_ordenados(lista){
    let ordenados = [];

    for (let i=0; i < lista.length; i++){
        for (let j=i + 1; j < lista.length; j++){
            ordenados.push([lista[i], lista[j]]);
        }
    }

    return ordenados;

}

lista = [1, 2, 3, 4]
console.log(pares_ordenados(lista));
